list/list_test.go

```diff
@@ -63,105 +84,192 @@ func (suite *TestSuite) TestFunctionality() {
 		o.EndpointOptions.DisableHTTPS = true
 	})
 
+	// Create config file
+	suite.configPath = filepath.Join(suite.tempDir, "s3cmd.conf")
+
+	// Write config file
+	if err = os.WriteFile(suite.configPath, []byte(fmt.Sprintf(configFormat, accessToken, suite.s3HTTPServer.URL)), 0600); err != nil {
```

Comment:
Might be better with 

```suggestion
	if err = os.WriteFile(suite.configPath, fmt.Appendf(nil, configFormat, accessToken, suite.s3HTTPServer.URL), 0600); err != nil {
```